java.lang.reflect.InvocationTargetException on Server Start

Full Stack Trace: 
```java
java.lang.reflect.InvocationTargetException
[12:23:55 WARN]: 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:116)
[12:23:55 WARN]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
[12:23:55 WARN]: 	at Honey-1.2.0a-all.jar//io.github.thewebcode.honey.event.Event.call(Event.java:12)
[12:23:55 WARN]: 	at Honey-1.2.0a-all.jar//io.github.thewebcode.honey.event.PacketEventListener.onLanguageUpdatePacket(PacketEventListener.java:42)
[12:23:55 WARN]: 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
[12:23:55 WARN]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
[12:23:55 WARN]: 	at Honey-1.2.0a-all.jar//io.github.thewebcode.honey.netty.event.InvokableEventMethod.invoke(InvokableEventMethod.java:46)
[12:23:55 WARN]: 	at Honey-1.2.0a-all.jar//io.github.thewebcode.honey.netty.event.RegisteredPacketSubscriber.invoke(RegisteredPacketSubscriber.java:52)
[12:23:55 WARN]: 	at Honey-1.2.0a-all.jar//io.github.thewebcode.honey.netty.event.PacketEventRegistry.invoke(PacketEventRegistry.java:30)
[12:23:55 WARN]: 	at Honey-1.2.0a-all.jar//io.github.thewebcode.honey.netty.handler.PacketChannelInboundHandler.channelRead0(PacketChannelInboundHandler.java:19)
[12:23:55 WARN]: 	at Honey-1.2.0a-all.jar//io.github.thewebcode.honey.netty.handler.PacketChannelInboundHandler.channelRead0(PacketChannelInboundHandler.java:9)
[12:23:55 WARN]: 	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
[12:23:55 WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
[12:23:55 WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
[12:23:55 WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
[12:23:55 WARN]: 	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346)
[12:23:55 WARN]: 	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318)
[12:23:55 WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
[12:23:55 WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
[12:23:55 WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
[12:23:55 WARN]: 	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
[12:23:55 WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)
[12:23:55 WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
[12:23:55 WARN]: 	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
[12:23:55 WARN]: 	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
[12:23:55 WARN]: 	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
[12:23:55 WARN]: 	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
[12:23:55 WARN]: 	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
[12:23:55 WARN]: 	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
[12:23:55 WARN]: 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
[12:23:55 WARN]: 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[12:23:55 WARN]: 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[12:23:55 WARN]: 	at java.base/java.lang.Thread.run(Thread.java:1623)
[12:23:55 WARN]: Caused by: java.lang.NullPointerException: Cannot invoke "org.bukkit.entity.Player.getUniqueId()" because "player" is null
[12:23:55 WARN]: 	at Honey-1.2.0a-all.jar//io.github.thewebcode.honey.event.EventListener.onPlayerLanguageUpdate(EventListener.java:46)
[12:23:55 WARN]: 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
```

PlayerLanguageUpdateEvent gets called to early

--- a/Honey/src/main/java/io/github/thewebcode/honey/event/PacketEventListener.java
+++ b/Honey/src/main/java/io/github/thewebcode/honey/event/PacketEventListener.java
@@ -39,6 +39,7 @@ public class PacketEventListener {
             Player player = Bukkit.getPlayer(UUID.fromString(packet.getSenderUUID()));
             HoneyUpdateLanguageSettingC2SPacket.Language language = packet.getLanguage();
 
+            if (player == null) return;
             new PlayerUpdateLanguageEvent(player, language).call();
         }
     }

